ui/react-ui/src/components/routes/userTasks/UserTasks.js

```diff
@@ -1,34 +1,48 @@
 import React, {useState, useEffect} from 'react';
 import Container from 'react-bootstrap/esm/Container';
-import {useParams} from "react-router-dom";
+import {useParams, useHistory} from "react-router-dom";
 import BootstrapTable from '../../bootstrapTable/BootstrapTable.js';
+import Button from 'react-bootstrap/Button';
+
+
 
 var UserTasks = () => {
-  const [userTasks, setUserTasks] = useState([]);
+  var [userTasks, setUserTasks] = useState(['loading']);
 
   var {userId} = useParams();
+  const history = useHistory();
 
   useEffect(() => {
-    // url should be replaced with => https://nsc-functionsapp-team1.azurewebsites.net/api/${userId}/tasks?
-    fetch(`https://w21httptriggernataliatest.azurewebsites.net/api/${userId}/tasks?`)
+    setUserTasks(['loading']);
+
+    fetch(`https://nsc-functionsapp-team1.azurewebsites.net/api/users/${userId}/tasks?`)
```

Comment:
Endpoint is updated with the correct url. Good work!

Reply:
^Same comment from me!

Glad to know that the `tasks` endpoint is working. I just changed to the endpoint of this URL for the UserID endpoint, which is also working now.